app/src/main/java/co/ostorlab/insecure_app/bugs/calls/BiometricFingerprintManagerVulnerability.java

```diff
@@ -0,0 +1,55 @@
+package co.ostorlab.insecure_app.bugs.calls;
+
```

Comment:
Missing unit test.

Reply:
Added unit test, 
for the other classes i need to add them first to the android app.